client/src/views/owner-home.js

```diff
@@ -0,0 +1,107 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import { Title } from "../components/ui";
+import OwnerSportFieldCard from "../components/owner-sportfield-card";
+import OwnerClubCircle from "../components/owner-club-circle";
+import OwnerCreateButton from "../components/owner-create-button";
+import { useSetClubs, useSetSportFields } from "../actions/action-hooks";
+import { useClubs, useSportFields } from "../selectors/selectors";
+import { getClubs } from "../services/club";
+import { getSportFields } from "../services/sportField";
+
+function OwnerHome() {
+  const clubs = useClubs();
+  const sportFields = useSportFields();
+  const setClubs = useSetClubs();
+  const setSportFields = useSetSportFields();
+  const [activeClub, setActiveClub] = React.useState(1);
```

Comment:
I think the default value should be null, because if there are no clubs, it means that there are no sport fields either and when you make the request to get the sports fields it will give error:
```
const [activeClub, setActiveClub] = React.useState(null);
```